robot_ws/src/cloudwatch_robot/launch/deploy_await_commands.launch

```diff
@@ -0,0 +1,17 @@
+<launch>
```

Comment:
From the description of this file, it seems like it has the same purpose as the `await_commands.launch` that we currently have (https://github.com/aws-robotics/aws-robomaker-sample-application-cloudwatch/blob/master/robot_ws/src/cloudwatch_robot/launch/await_commands.launch). 

I'm not seeing the need to have two of these files. @samuelgundry Should the original `await_commands.launch` launch file contain a line to bringup the turtlebot? Is it just an oversight that it currently does not.

Reply:
Yep, deploy_await_commands.launch should include await_commands.launch.   We should also document why we have an additional .launch file - the RoboMaker Deploy API does not accept a command-line argument, so we can't have a .launch argument to turn on/off turtlebot3_bringup

Reply:
See Hello World for example: https://github.com/aws-robotics/aws-robomaker-sample-application-helloworld/blob/master/robot_ws/src/hello_world_robot/launch/deploy_rotate.launch

Reply:
Make sense, I will use the await_commands.launch in the new launch file.

Reply:
> We should also document why we have an additional .launch file - the RoboMaker Deploy API does not accept a command-line argument, so we can't have a .launch argument to turn on/off turtlebot3_bringup

@samuelgundry do you think the current comments on the script and launch files are enough, of should be add something to the README as part of this PR?